README.md

```diff
@@ -63,16 +63,24 @@ const thx = msgFV["no"];
 
 **Cluster**
 
+First, if you need to set up nodes into a working redis cluster:
```

Comment:
Could you run `deno fmt README.md` locally and push this file again?

Reply:
Doh, should've done that. Just pushed up. Let me know if you'd like me to squash these, or you can just squash at your leisure. :)